paddle/fluid/operators/softmax_op.cc

```diff
@@ -115,6 +115,11 @@ class SoftmaxOpMaker : public framework::OpProtoAndCheckerMaker {
     AddAttr<bool>("use_mkldnn",
                   "(bool, default false) Only used in mkldnn kernel")
         .SetDefault(false);
+    AddAttr<std::string>(
+        "mkldnn_data_type",
+        "(string, default \"float32\"). Data type of mkldnn kernel")
+        .SetDefault("float32")
+        .InEnum({"float32", "int8", "bfloat16"});
```

Comment:
I'm not sure softmax int8 will ever exist. Won't there be any risk here that softmax may be tried to be initialzied as int8?

Reply:
Yes you right, I will change it.